recipe/meta.yaml

```
@@ -21,26 +21,36 @@ build:
   script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir --no-build-isolation -vvv
 
 requirements:
+  build:
+    # we use go directly because we do not have activation feedstocks
+    # therefore we can't use compiler('go') and we skip linting
+    # for should_use_compilers
+    - go
+    - {{ compiler('c') }}
+    - {{ compiler('rust') }}
+
   host:
     - pip
-    - wheel
-    - python 
+    - python
+    - hatchling
     - setuptools
-    
+    - typing-extensions 4.7
```

```suggestion
    - typing-extensions
```
You can drop pinnings for Python packages in `host`, see the `recipe standards` https://github.com/anaconda-distribution/perseverance-skills/blob/main/sections/02_Package_building/02_Recipes/Recipe_standards.md#the-requirements-section
It will simplify future Python build-outs

---

That's a fix for lint error: "recipe/meta.yaml:37: host_section_needs_exact_pinnings: Linked libraries host should have exact version pinnings."

---

Python packages are the exception, but if a Python package relies on C API (`numpy`, `scipy`, etc.) then you need to pin it in `host`.